pkg/azclient/client-gen/generator/template.go

```diff
@@ -72,7 +72,7 @@ var CreateOrUpdateFuncTemplate = template.Must(template.New("object-scaffolding-
 {{- end }}
 const CreateOrUpdateOperationName = "{{.ClientName}}.Create"
 // CreateOrUpdate creates or updates a {{$resource}}.
-func (client *Client) CreateOrUpdate(ctx context.Context, resourceGroupName string, resourceName string,{{with .SubResource}}parentResourceName string, {{end}} resource {{.PackageAlias}}.{{$resource}}) (result *{{.PackageAlias}}.{{$resource}}, err error) {
+func (client *Client) CreateOrUpdate(ctx context.Context, resourceGroupName string,{{if .SubResource}}parent{{.Resource}}Name string, {{end}} target{{$resource}}Name string, resource {{.PackageAlias}}.{{$resource}}) (result *{{.PackageAlias}}.{{$resource}}, err error) {
```

Comment:
remove parent and target word, make the parameter naming more straightfoward

Reply:
fixed. PTAL. Thanks! @andyzhangx